src/NJsonSchema/Generation/JsonSchemaGenerator.cs

```diff
@@ -1089,7 +1089,7 @@ private void GenerateInheritanceDiscriminator(Type type, JsonSchema schema, Json
             }
         }
 
-        private object? TryGetInheritanceDiscriminatorConverter(Type type)
+        private SystemTextJsonInheritanceWrapper? TryGetInheritanceDiscriminatorConverter(Type type)
```

Comment:
is this really needed? 
i think this might break CI?

Reply:
Yes, I think this should be reverted, it breaks test `When_schema_has_base_schema_then_it_is_referenced_with_Newtonsoft`. 

However, without it build fails with " [ERR] Compile: D:\a\NJsonSchema\NJsonSchema\src\NJsonSchema\Generation\JsonSchemaGenerator.cs(1092,25): error CA1859: Change return type of method 'TryGetInheritanceDiscriminatorConverter' from 'object?' to 'NJsonSchema.Generation.JsonSchemaGenerator.SystemTextJsonInheritanceWrapper?' for improved performance (https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1859) [D:\a\NJsonSchema\NJsonSchema\src\NJsonSchema\NJsonSchema.csproj::TargetFramework=netstandard2.0]"."

Reply:
Can you suggest a better fix to CA1859?

Reply:
You should be able to use 

```c#
#pragma warning disable CA1859
private object? TryGetInheritanceDiscriminatorConverter(Type type)
#pragma warning enable CA1859
```